public/components/Question/UpdateQuestion/update_question.js

```diff
@@ -48,29 +62,29 @@ export const createQuestionUpdate = (question) => {
     default:
       textButton.checked = true;
       renderAnswers();
-      buttonContainer.style.display = 'none';
+      buttonAddAnswer.style.display = 'none';
       break;
   }
 
   radioButton.addEventListener('click', () => {
     checkboxButton.checked = false;
     textButton.checked = false;
     type = 1;
-    buttonContainer.style.display = 'flex';
+    buttonAddAnswer.style.display = 'flex';
```

Comment:
давайте в очередной итерации этого кода (если сейчас сложно / долго править) - замените залазинье в style на добавление css класса - особенно с кейсом когда что-то `display: none` 

\+ лучше если уж так и оставлять, то заменить здесь / выше / ниже на
```suggestion
    buttonAddAnswer.style.display = null;
```
вместо того чтобы flex задавать - а вдруг не flex =)

https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/style

Reply:
уже напарывались, прямо в этом примере - и действительно, не флекс

Reply:
добавил в TODO